lib/taiko.js

```diff
@@ -2245,7 +2261,7 @@ for(const func in module.exports){
 }
 
 module.exports.metadata = {
-    'Browser actions': ['openBrowser', 'closeBrowser', 'client', 'switchTo', 'setViewPort', 'openTab', 'closeTab', 'overridePermissions', 'clearPermissionOverrides', 'setCookie', 'clearBrowserCookies', 'deleteCookies'],
+    'Browser actions': ['openBrowser', 'closeBrowser', 'client', 'switchTo', 'setViewPort', 'openTab', 'closeTab', 'overridePermissions', 'clearPermissionOverrides', 'setCookie', 'clearBrowserCookies', 'deleteCookies', 'currentURL'],
```

Comment:
This is more a Page action as it gives the url of current page. Can this be moved to page actions ?

Reply:
@jaydeepc I think this change is pending.